app/config.py

```diff
@@ -12,3 +12,36 @@ def __init__(self, config_file="config/config.json"):
         with open(config_file, "rt") as conf:
             self.__dict__ = json.loads(conf.read())
 
+    subscription_id = ""
```

Comment:
Why do you need it here? We have all of these in json file.
I don't want to have 2 places with duped info.

Reply:
I agree that we don't want to have things defined in two places because it becomes a maintenance nightmare. However, the reason why I am adding the class attributes here are for the following:
1. intellisense - when we use the config class, it is nice to see what values we have access to
2. default values - if you don't have a config value in config.json, it won't be overridden when we instantiate Config.